File: FAANG/findTwoNums.py
def findTwoNums(nums, target):
    for i in range(len(nums)):
        ntf = target - nums[i]
        j = i + 1
        for j in range(len(nums)):
            if nums[j] is ntf:
                return [i, j]
    return 'cannot find answer'

print(findTwoNums([1, 3, 7, 9, 4], 4))

File: FAANG/findTwoNums.php
<?php
function findTwoNums(array $nums, int $target){
    for ($i=0; $i < count($nums); $i++) { 
        $numToFind = $target - $nums[$i];
        for ($j=$i + 1; $j < count($nums); $j++) { 
            if ($nums[$j] == $numToFind) {
               return [$i, $j];
            }
        }
    }
    return null;
}

print(findTwoNums([1, 3, 7, 9, 2], 11));
